src/engraving/dom/chord.cpp

```diff
@@ -158,6 +158,24 @@ Note* Chord::downNote() const
     return result;
 }
 
+//---------------------------------------------------------
+//   withUpNoteDistance
+//---------------------------------------------------------
+
+int Chord::withUpNoteDistance(int inPitch) const
```

Comment:
From the name, I don't understand what this method does (I don't know very well either how to interpret the name at all). So:
- What does it do?
- How was the name supposed to explain that?
- Can we find a better name that explains that even better?